src/components/Person/Person.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = (props) => {
+  const {
+    name,
+    age = null,
+    sex,
+    isMarried = false,
+    partnerName = null,
+  } = props;
+
+  const partner = sex === 'f' ? `husband's` : `wife's`;
+  const message = isMarried
+    ? `My ${partner} name is ${partnerName}`
+    : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {'My name is '}
+        {name}
```

Comment:
Let's use intepolation for better readability 
```suggestion
        {`My name is ${name}`}
```